src/api.ts

```diff
@@ -65,3 +67,49 @@ export interface INCHIOutput {
   szMessage: string;
   szLog: string;
 }
+
+export function GetStructFromINCHI(input: string,options: string[] = [""]):0 | -1 | -2 | 1 | 2 | 3 | 4 | 5 {
```

Comment:
Declare `0 | -1 | -2 | 1 | 2 | 3 | 4 | 5` as a union type and then reuse it.

Do the same for `0 | -1 | 1 | 2 | 3 | 4` and `0 | -1 | 1 | 2 | 3`.